OutOfSchool/Tests/OutOfSchool.Tests.Common/TestDataGenerators/WorkshopBaseDtoGenerator.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Bogus;
+using OutOfSchool.Common.Enums;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Models.Workshops;
+using System;
+using System.Collections.Generic;
+
+namespace OutOfSchool.Tests.Common.TestDataGenerators;
+
+public static class WorkshopBaseDtoGenerator
+{
+    private static readonly Faker<WorkshopBaseDto> Faker = new Faker<WorkshopBaseDto>()
+        .RuleForType(typeof(int), f => f.Random.Int())
+        .RuleForType(typeof(Guid), f => f.Random.Guid())
+        .RuleForType(typeof(long), f => f.Random.Long(0, long.MaxValue))
+        .RuleForType(typeof(string), f => f.Lorem.Word())
+        .RuleFor(x => x.Id, f => f.Random.Guid())
+        .RuleFor(x => x.Title, f => f.Company.CompanyName())
+        .RuleFor(x => x.Phone, f => f.Phone.ToString())
+        .RuleFor(x => x.Email, f => f.Person.Email)
+        .RuleFor(x => x.Website, f => f.Internet.Url())
+        .RuleFor(x => x.Facebook, f => f.Person.UserName)
+        .RuleFor(x => x.Instagram, f => f.Person.UserName)
+        .RuleFor(x => x.MinAge, f => f.Random.Number(1, 15))
+        .RuleFor(x => x.MaxAge, f => f.Random.Number(16, 18))
+        .RuleFor(x => x.DateTimeRanges, f => f.Make(new Random().Next(1, 4), () =>
+            new DateTimeRangeDto()
+            {
+                Id = f.Random.Long(),
+                StartTime = f.Date.Timespan(),
+                EndTime = f.Date.Timespan(),
+                Workdays = new List<DaysBitMask>()
+            }))
```

Comment:
(Code duplicates) Please consider extracting bit more generators.
This (`() => new DateTimeRangeDto() { ... }`) part of code is repeated in subsequent generators.
Same (at least) for `WorkshopDescriptionItemDto` and `AddressDto`.

Reply:
Fixed it. The additional test data generators were added.